diff --git a/test/images/image-util.sh b/test/images/image-util.sh
index 9b729b34..d45f1eb3 100755
--- a/test/images/image-util.sh
+++ b/test/images/image-util.sh
@@ -34,7 +34,7 @@ source "${KUBE_ROOT}/hack/lib/logging.sh"
 source "${KUBE_ROOT}/hack/lib/util.sh"
 
 # Mapping of go ARCH to actual architectures shipped part of multiarch/qemu-user-static project
-declare -A QEMUARCHS=( ["amd64"]="x86_64" ["arm"]="arm" ["arm64"]="aarch64" ["ppc64le"]="ppc64le" ["s390x"]="s390x" )
+declare -A QEMUARCHS=( ["amd64"]="x86_64" ["arm"]="arm" ["arm64"]="aarch64" ["ppc64le"]="ppc64le" ["s390x"]="s390x" ["loong64"]="loongarch64" )
 
 # NOTE(claudiub): In the test image build jobs, this script is not being run in a git repository,
 # which would cause git log to fail. Instead, we can use the GIT_COMMIT_ID set in cloudbuild.yaml.
@@ -90,7 +90,7 @@ getBaseImage() {
 # This function will build test image for all the architectures
 # mentioned in BASEIMAGE file. In the absence of BASEIMAGE file,
 # it will build for all the supported arch list - amd64, arm,
-# arm64, ppc64le, s390x
+# arm64, ppc64le, s390x, loong64
 build() {
   local image=${1}
   local img_folder=${1}
